blog/01-18-2023-pros-and-cons-of-celo-to-create-decentralized-applications

```diff
@@ -0,0 +1,109 @@
+---
+title: Pros and Cons of using Celo to create Decentralized Applications
+description: Pros and Cons of using Celo to create Decentralized Applications
+authors:
+  - name: Maxwell Onyeka
+    title: Technical writer
+    image_url: https://avatars.githubusercontent.com/u/80678178?v=4
+tags: [celosage, beginner, foundation]
+hide_table_of_contents: true
+slug: /tutorials/pros-and-cons-of-using-celo-to-create-decentralized-applications
+---
+
+![header](../../src/data-tutorials/showcase/beginner/pros.png)
```

Comment:
update your directory (eliminate "../..")```suggestion
![header](/src/data-tutorials/showcase/beginner/pros.png)
```

Reply:
It will affect the image